.github/issue_template.md

```diff
@@ -0,0 +1,36 @@
+<!---
+ Please use this template to fill your issue, issues that does not
+ follow this will be asked to be edited to match the template, or will be closed
+
+ We use github issues for problem/bug reports, for questions, please use the Discord channel, the link for it can be found on the project README
+-->
+
+## Description
+
+Describe your issue here
+
+## Development enviroment
+
+Flame version:
+
+<!--
+NOTE: Flame supports stable and beta channel of Flutter, issues out of those channels may not be prioritized, or closed.
```

Comment:
```suggestion
NOTE: Flame supports the stable and beta Flutter channels; issues outside of those channels may not be prioritized, or closed.
```

Reply:
I would use full stop instead of semi-colon.

Reply:
both work for me :)